src/features/device/pca10171.json

```
@@ -83,7 +83,7 @@
             {
                 "name": "nRF Cloud multi-service",
                 "type": "jlink",
-                "description": "Evaluate nRF Cloud services: onboarding, data collection, FOTA updates, logging, and alerts.",
+                "description": "Evaluate nRF Cloud services: Onboarding, data collection, FOTA updates, location, logging and alerts.",
```

```suggestion
                "description": "Evaluate nRF Cloud services: Onboarding, data collection, FOTA updates, location, logging, and alerts.",
```
Why is onboarding capitalized?

---

Due to a request,
I'm more inclined to accept whatever you decide but I can pass it on just to check whether there was any explicit reason for it.